app/controllers/menus_controller.rb

```diff
@@ -0,0 +1,44 @@
+class MenusController < ApplicationController
+  def index
+    @menus = Menu.all
+  end
+
+  def show
+    @menu = Menu.find(params[:id])
+
+    @order = Order.new
+    @order.menu_id = @menu.id
+
+    @meal = Meal.new
+    @meal.menu_id = @menu.id
+  end
+
+  def new
+    @menu = Menu.new
+  end
+
+  def create
+    @menu = Menu.new
+    @menu.save
+    flash.notice = 'Menu created!'
+    redirect_to menu_path(@menu)
```

Comment:
Just FYI. You can write one-liners for [redirect_to with flash](https://apidock.com/rails/ActionController/Base/redirect_to)

Reply:
like `redirect_to <path>, notice: <message>`